tasks/watch.cr

```diff
@@ -15,11 +16,13 @@ module Sentry
     getter app_processes = [] of Process
     property successful_compilations
     property app_built
+    property reload_browser
```

Comment:
If you do `property?` you can then call `reload_browser?` which reads a bit better IMO

Reply:
That's neat, it definitely reads better